app/models/user.rb

```diff
@@ -29,6 +29,7 @@ class User < ApplicationRecord
   validates :role, inclusion: { in: ROLES }
 
   scope :department_assocable, (-> { where(role: CAN_MANAGE_DEPARTMENT) })
+  scope :where_superadmin, (-> { where(role: SUPERADMIN) })
```

Comment:
I would just call this scope `superadmins`

Reply:
I named it like that in order to have it clearly distinguishable from User.superadmin? and such. But actually, right...